src/script/signingprovider.cpp

```diff
@@ -82,13 +87,21 @@ bool FlatSigningProvider::GetTaprootBuilder(const XOnlyPubKey& output_key, Tapro
     return LookupHelper(tr_trees, output_key, builder);
 }
 
+std::vector<CPubKey> FlatSigningProvider::GetAggregateParticipantPubkeys(const CPubKey& pubkey) const
+{
+    const auto& it = aggregate_pubkeys.find(pubkey);
+    if (it == aggregate_pubkeys.end()) return {};
+    return it->second;
```

Comment:
nit: could use `LookupHelper` here (for slightly better readability imho)
```suggestion
    std::vector<CPubKey> participant_pubkeys;
    LookupHelper(aggregate_pubkeys, pubkey, participant_pubkeys);
    return participant_pubkeys;
```

Reply:
Done